core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -4386,7 +4394,7 @@ public void run() {
     @CLIMethod(name="safe-shutdown")
     @RequirePOST
     public HttpResponse doSafeExit(StaplerRequest req) throws IOException {
-        checkPermission(ADMINISTER);
+        checkPermission(MANAGE);
```

Comment:
can the MANAGER user recover from this? (could they restart it). its not a security issue theres no leak but just wondering if they should be allowed to do this (what would the use case be)